python/cudnn/gemm/frost/kernel_templates/sm100_matmul.py

```
@@ -469,6 +525,11 @@ def _kernel(
 
             sched_iter += 1
 
+        if cutlass.const_expr(USE_PDL and split_k_slices > 1):
```

Ruff (0.16.3)

[error] Undefined name `split_k_slices`

(F821)